File: Level3.rb
startingArray = [ 'ab123', 'gh00', 'ijk8', 'lmn12', 'cd99ef11' ]
numbersArray = []

puts startingArray

puts "If we dump the letters in the elements of the above array, we get......."

for i in startingArray
    numbersArray.push(i.delete("^0-9"))
end

numbersArray.pop

numbersArray.push('99')
numbersArray.push('11')

puts numbersArray

puts "And now, we'll increase each number by 1...."

for i in numbersArray
    i.map(&:next)
end

File: Javascript/Level3.js
input = ['ab123', 'gh00', 'ijk8', 'lmn12', 'cd99ef11']
separator = []
numbers = []
letters = []
alert(input);
alert("if we separate the numbers from the letters, add 1 to each of those numbers, and then rejoin the numbers and letters, we get.......");

for ( i=0; i < input.length; i++ ) {
    separator.push(input[i]);
    for ( x = 0; x < separator.length; x++ ) {
        alert(separator.charAt(x));
        if (isNaN(separator[x]) == true) {
            letters.push(separator[x]);
        } else {
            numbers.push(separator[x]);
        };
    };
    separator.length = 0;
};
alert(numbers);
alert(letters);